Software/Rig/Rig.ino

```diff
@@ -73,17 +76,34 @@ void setup() {
   pinMode(PIN_TONE_POSITIVE, OUTPUT);
   pinMode(PIN_BUTTON, INPUT_PULLUP);
   pinMode(PIN_LICK, INPUT);
+
+  // Each session of trials has a different random seed taken by
+  // reading from a disconnected pin.
+  randomSeed(analogRead(0));
+
+  // Each session is a random permutation of {CS-, CS-, CS-, CS+, CS+, CS+}.
+  // Note that 0 is CS-, 1 is CS+. This uses Fisher-Yates shuffle. Note
+  // that this assumes there are `6` trials and would not fully randomize
+  // the trials otherwise
+  for (int i = NUMBER_OF_TRIALS-1; i > 0; --i) {
```

Comment:
This "works" but seems like it has a bias towards starting trials at CS+. It could be because of the random seed using an analog read vs. some other source of randomness.